README.md

```diff
@@ -71,9 +71,22 @@ A list of limitations is present in the [issues](https://github.com/availproject
 
 If you feel some important limitations are not covered, please check out our [Contributing section](https://github.com/availproject/op-evm#contributing) and open an issue or a PR. 
 
-## Contributing
+## Contribution Guidelines
+
+### Rules
+
+Avail welcomes contributors from anywhere and from any kind of education or skill level. We strive to create a community of developers that is welcoming, friendly and right. We welcome contributions to the OpEVM project. If you find any issues, have suggestions for improvements, or would like to contribute new features, please open a GitHub issue or submit a pull request.
+
+1. Before asking any questions regarding how the project works, please read through all the documentation and install the project on your own local machine to try it and understand how it basically works. Please ask your questions in open channels (Github and TG).
```

Comment:
```suggestion
1. Before raising a question, please take the initiative to read the [official documentation](https://docs.availproject.org/). We recommend installing the project on your local machine to get a hands-on understanding. If questions arise, feel free to ask in our open channels on [GitHub](https://github.com/orgs/availproject/discussions) and Telegram.
```

Reply:
In the case of op-evm contributions doesn't it make sense to ask to look the project's documentation?